src/random.cr

```diff
@@ -373,34 +373,6 @@ module Random
     random_bytes(n).hexstring
   end
 
-  # Generates a UUID (Universally Unique Identifier).
```

Comment:
Why there's no replacement of `Random#uuid`?
It's a handy method, why was it removed?

Reply:
...because `UUID.new` doesn't use the random instance you're calling `#uuid` on, it always uses `Random::Secure`. And for good reason.